core/src/main/java/com/exadel/aem/backpack/core/dto/response/JcrPackageWrapper.java

```diff
@@ -0,0 +1,68 @@
+package com.exadel.aem.backpack.core.dto.response;
```

Comment:
Please add license info(search in other files)

Reply:
done